clean.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: clean [options]"
+    echo "Cleans the local dev environment."
+    echo
+    echo "  -b         Delete bin directory"
+    echo "  -p         Delete packages directory"
+    echo "  -t         Delete Tools directory"
+    echo "  -w         Remove all untracked files in the working tree, including the bin, packages, and Tools directories"
```

Comment:
Looking at issue #5513 and the dev workflow we have internally we should use /s instead of /w